drf_spectacular/openapi.py

```diff
@@ -271,11 +271,11 @@ def get_auth(self):
             auths.append({})
         return auths
 
-    def get_request_serializer(self):
+    def get_request_serializer(self) -> serializers.BaseSerializer:
```

Comment:
Given that `_get_serializer` can return `None`, the type would need to be `Optional[]`. But even that is not really sufficient.

for the base functionality this is adequate, but through `extend_schema` this may be a heap of different things (e.g. differently keyed dicts, basic types, enums, etc). that's the reason why `extend_schema` has 

```python
        request: Any = empty,
        responses: Any = empty,
 ```

i would prefer having no hint or `Any` instead of a misleading incomplete type hint.

Reply:
Good point. `Any` will work just fine for me, if you're good with it!